.pipelines/pipeline.user.windows.buddy.yml

```diff
@@ -0,0 +1,42 @@
+version:
+  major: 0
+  minor: 0
+  tag: 'SNAPSHOT'
+  name: 'ApplicationInsights-Java'
+  system: 'custom'
+build:
+  commands:
+    - !!buildcommand
+      name: 'Assemble Perf Counters DLL'
+      command: '.scripts/gradle.cmd'
+      arguments: ':core:windowsSharedLibrary'
+      artifacts:
+        - from: 'core/build/libs/windows/shared'
+          include:
+            - '**/*'
+          exclude:
+            - '**/*.pdb'
+          signing_options:
+            profile: external_distribution
+            sign_inline: true

```

Comment:
can the buddy builds sign things?

Reply:
it was signing 3rd party dll, it used external_distribution.  it works on the buddy build though.. i didn't find any errors so far.

Reply:
i can remove this.. signing buddy build doesn't offer much benefits.

Reply:
it will save some build time :)